UNITTESTS/stubs/EventFlags.cpp

```diff
@@ -0,0 +1,24 @@
+#include "rtos/EventFlags.h"
```

Comment:
What are the rules for `stubs/` directory? It's generic so I'm guessing only "empty" stubs, like below, are allowed? If I need something specific to be done I need to create specific one in test directory? Can we document it somewhere?

Reply:
This stub file does not have any logic so it looks useless. Usually a stub can support all unittests, module tests and even system tests if done correctly. I personally collect all stubs into stubs/ folder and use them there.
If you need a stub which cannot be reused by other tests then it should be in test directory.

Reply:
Stubs that require them to do something, can have internal parameters that tests can configure. But usually those are reusable by other tests as well.

Then some stubs tend to be just `return 0;`

Reply:
At the end when all classes have tests, all stubbed functions should have some internal implementation. Otherwise the real function can always return 0 and would be having useless return value!